recipes-ni/ni-grpc-device/ni-grpc-device_1.0.1.bb

```diff
@@ -0,0 +1,40 @@
+SUMMARY = "gRPC server providing remote access to NI device driver APIs."
+DESCRIPTION = ""
+HOMEPAGE = "https://github.com/ni/grpc-device"
+SECTION = "base"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "\
+	https://github.com/ni/grpc-device/releases/download/${${PN}_RELEASE_TAG}/ni-grpc-device-server-ni-linux-rt-x64.tar.gz;name=release-tar;downloadfilename=${PN}_${${PN}_RELEASE_TAG}.tar.gz;subdir=${PN}-server \
+	https://github.com/ni/grpc-device/releases/download/${${PN}_RELEASE_TAG}/ni-grpc-device-client.tar.gz;name=release-client-tar;downloadfilename=${PN}-client_${PN}_${${PN}_RELEASE_TAG}.tar.gz;subdir=${PN}-client \
+"
+
+SRC_URI[release-tar.md5sum] = "200de2f8b1cbc4c26af42cfaed1b01ea"
+SRC_URI[release-tar.sha256sum] = "32e0767b28f71a3024921904aa7bcd879cf9e9f2a98661722ac549446bca0aed"
+SRC_URI[release-client-tar.md5sum] = "11da718d6be42848a1e4155e56bdb641"
+SRC_URI[release-client-tar.sha256sum] = "0555fc1f2867b4ed9314354ed87041b7065365a24ddaf3526c1871f519df34ad"
+
+${PN}_RELEASE_TAG ?= "v${PV}"
+${PN}_RELEASE_TAG = "v${PV}-rc1"
+
+S = "${WORKDIR}"
+
+DEPENDS += "\
+"
+
+RDEPENDS_${PN} += "\
+	python3 \
+"
+
+do_install_append () {
```

Comment:
What are you appending to here?

Reply:
The `do_install` task, which moves content from the source directories into the staging rootfs (which is then split into packages). Normally, this task can be automatically fulfilled by a project's `make install`. In this case, we're building the packages manually using already-compiled source.

Reply:
I guess in the past I have seen `_append` when the recipe is a bbappend or inherits from a class that already has do_install() defined. In this case, this is a new recipe and I wasn't sure if there was a default install definition that you are appending to. Given that the recipe works as is, I suppose that's the case.

Reply:
Yeah; I believe `package.bbclass` implements a default (blank) `do_install`. I *think* I could technically use `do_install ()` here and replace it whole-sale; but I think this is conceptually correct.